README.md

```diff
@@ -33,6 +33,14 @@ After using [CyberChef](https://gchq.github.io/CyberChef/) to experiment and fin
 - In a browser, navigate to `localhost:3000` to see usage documentation.
 
 
+### Docker
+A Docker image can be built, then run by doing the following:
+
+- `git clone https://github.com/gchq/CyberChef-server`
+- `cd CyberChef-server`
+- `sudo docker build -t mylocalrepo/cyberchef-server .`
```

Comment:
By default people don't need to run docker with sudo. To avoid confusion, please could you remove the `sudo` from these examples?

Reply:
Could you also remove `mylocalrepo` from the docker commands for brevity please?

Reply:
Sure thing!

Reply:
This has been completed.